source/backend/tests/unit/api/Controllers/Leases/ConsultationControllerTest.cs

```diff
@@ -0,0 +1,204 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using FluentAssertions;
+using MapsterMapper;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using Pims.Api.Areas.Disposition.Controllers;
+using Pims.Api.Areas.Lease.Controllers;
+using Pims.Api.Helpers.Exceptions;
+using Pims.Api.Models.Concepts.DispositionFile;
+using Pims.Api.Models.Concepts.Lease;
+using Pims.Api.Services;
+using Pims.Core.Test;
+using Pims.Dal.Entities;
+using Pims.Dal.Exceptions;
+using Pims.Dal.Security;
+using Xunit;
+
+namespace Pims.Api.Test.Controllers
+{
+    [Trait("category", "unit")]
+    [Trait("category", "api")]
+    [Trait("group", "consultation")]
+    [ExcludeFromCodeCoverage]
+    public class ConsultationControllerTest
+    {
+        #region Variables
+        private Mock<ILeaseService> _service;
+        private ConsultationController _controller;
+        private IMapper _mapper;
+        #endregion
+
+        public ConsultationControllerTest()
+        {
+            var helper = new TestHelper();
+            this._controller = helper.CreateController<ConsultationController>(Permissions.LeaseEdit, Permissions.LeaseView);
+            this._mapper = helper.GetService<IMapper>();
+            this._service = helper.GetService<Mock<ILeaseService>>();
+        }
+
+        #region Tests
+        [Fact]
+        public void GetByConsultationId()
+        {
+            // Arrange
+            var consultation = new PimsLeaseConsultation() { LeaseId = 1 };
+
+            this._service.Setup(m => m.GetConsultationById(It.IsAny<long>())).Returns(consultation);
+
+            // Act
+            var result = this._controller.GetLeaseConsultationById(1, 1);
+
+            // Assert
+            this._service.Verify(m => m.GetConsultationById(It.IsAny<long>()), Times.Once());
```

Comment:
not a big fan of these tests. It leaks implementation details about what the controller does internally and does not provide a validation of anything other than proxying the call

Reply:
This is our pattern thus far for these tests, which we have gone through together as a group. Recommend we bring this up for review in our dev meeting, as nearly all of our controllers are tested this way.